### PR TITLE
Add Vatican City geocoding

### PR DESCRIPTION
**Pull request by bigfoot90**
This PR adds `+379` Vatican country code ([as Google says](https://www.google.it/search?q=vatican+prefix&oq=vatican+prefix&aqs=chrome..69i57j0l2.2655j0j7&sourceid=chrome&ie=UTF-8)),
and Vatican City geolocoding for `3906698`

### CODE REVIEW

**Comment by bigfoot90** [comment]
I signed it!

**Comment by penmetsaa** [comment]
Hi,

Thank you for taking the time to improve libphonenumber!

We presently do not accept pull requests that changes timezone metadata as it is auto generated. More details in: https://github.com/googlei18n/libphonenumber/blob/ebd7e1d348a4c771dcccf35caaf80cce7ffbfd80/resources/timezones/README.md#contributing-to-the-timezone-metadata

We will triage all the updates (including timezone) as part of issue: https://issuetracker.google.com/issues/111247429
So closing this PR.

**Comment by penmetsaa** [comment]
Re-opening this PR as it have geocoding updates. Please address the review comments. rest of the changes looks good to me.

File: resources/test/timezones/map_data.txt [hunk 1]
```diff
@@ -21,4 +21,5 @@
 1650960|America/Los_Angeles
 1989|Ameriac/Los_Angeles
 612|Australia/Sydney
+379|Europe/Vatican
```
[the remark this answers is not in the document]
Comment (bigfoot90) [reply resources/test/timezones/map_data.txt]: removed

**Comment by kkeshava** [comment]
Hi,
Thank you for contributing to libphonenumber! The changes look good to us :)

This PR is approved now. Please update your branch and once it is done successfully, we can merge this pull request.

**Comment by bigfoot90** [comment]
Solved by https://github.com/google/libphonenumber/pull/2232

**Pull request closed without merging**